server/sonar-process-monitor/src/main/java/org/sonar/application/cluster/AppStateClusterImpl.java

```diff
@@ -43,9 +43,13 @@
 import org.sonar.application.AppStateListener;
 import org.sonar.process.ProcessId;
 
-public class AppStateClusterImpl implements AppState {
-  static final String OPERATIONAL_PROCESSES = "operational_processes";
-  private static final String LEADER = "leader";
+import static org.sonar.application.config.SonarQubeVersionHelper.getSonarqubeVersion;
+
+public class AppStateClusterImpl implements AppState, AutoCloseable {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') "AutoCloseable" is already extended by "AppState"; there is no need to implement it here. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1939)